data.yaml

```diff
@@ -586,11 +586,121 @@ sections:
+      - [2, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Dismounter\">Dismounter</a>"]
+      - [3, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Bloodhound's+Fang\">Bloodhound's Fang</a>"]
+      - [4, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Magma+Wyrm's+Scalesword\">Magma Wyrm's Scalesword</a>"]
+      - [5, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Zamor+Curved+Sword\">Zamor Curved Sword</a>"]
+      - [6, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Omen+Cleaver\">Omen Cleaver</a>"]
+      - [7, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Monk's+Flameblade\">Monk's Flameblade</a>"]
+      - [8, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Beastman's+Cleaver\">Beastman's Cleaver</a>"]
+      - [9, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Morgott's+Cursed+Sword\">Morgott's Cursed Sword</a>"]
+  -
+    title: Curved Swords
+    link: "https://eldenring.wiki.fextralife.com/Curved+Swords"
+    id: curved_swords
+    num: 9
+    items:
+      - [1, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Falchion\">Falchion</a>"]
+      - [2, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Beastman's+Curved+Sword\">Beastman's Curved Sword</a>"]
+      - [3, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Shotel\">Shotel</a>"]
+      - [4, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Shamshir\">Shamshir</a>"]
+      - [5, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Bandit's+Curved+Sword\">Bandit's Curved Sword</a>"]
+      - [6, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Magma+Blade\">Magma Blade</a>"]
+      - [7, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Flowing+Curved+Sword\">Flowing Curved Sword</a>"]
+      - [8, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Wing+of+Astel\">Wing of Astel</a>"]
+      - [9, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Scavenger's+Curved+Sword\">Scavenger's Curved Sword</a>"]
+      - [10, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Eclipse+Shotel\">Eclipse Shotel</a>"]
+      - [11, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Serpent-God's+Curved+Sword\">Serpent-God's Curved Sword</a>"]
+      - [12, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Mantis+Blade\">Mantis Blade</a>"]
+      - [13, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Scimitar\">Scimitar</a>"]
+      - [14, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Grossmesser\">Grossmesser</a>"]
+      - [15, "f_weap", "<a href=\"https://eldenring.wiki.fextralife.com/Nox+Flowing+Sword\">Nox Flowing Sword</a>"]
   -
     title: Daggers
     link: "https://eldenring.wiki.fextralife.com/Daggers"
     id: daggers
-    num: 3
+    num: 10
```

Comment:
Once a `num` is in you cannot change it because it will screw up people's saved data. But num's don't have to be in order. So this one has to stay as 3, but you can just make bows 11.